src/utils/MigrationUtils.ts

```diff
@@ -0,0 +1,63 @@
+import { FillType, v2Deposit, v3Deposit, v2Fill, v3Fill, v2SpeedUp, v3SpeedUp } from "../interfaces";
```

Comment:
Is this file named `MigrationUtils` because it will be renamed or removed post-upgrade?

Reply:
Yeah, it was my intention to remove this once we remove the v2/2.5 functions from the contracts (onchain). Per James' comment below it seems like it might be useful to retain some or all of the functionality, so I guess we can decide whether to find a better name for it, or drop it from the SDK and relocate to the FE at that point.

Reply:
I think it might be better to rename it V3Utils or something

Reply:
Sure, will do.

Reply:
Done: https://github.com/across-protocol/sdk-v2/pull/495/commits/7901cb5c8b6082a70828192fd8c3c91cad6040f0